test/multiarch_utils.go

```diff
@@ -106,14 +118,44 @@ func imageNamesMapping() map[string]string {
 			"stedolan/jq":                           "ibmcom/jq-s390x:latest",
 			"gcr.io/kaniko-project/executor:v1.3.0": getTestImage(kanikoImage),
 		}
+	case "ppc64le":
+		return map[string]string{
+			"gcr.io/tekton-releases/dogfooding/alpine-git-nonroot:mario": "gcr.io/tekton-releases/dogfooding/alpine-git-nonroot:latest",
```

Comment:
This image is not used in tests any more

Reply:
Thanks for pointing out. Removed this unused mapping.